package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.2",
   "private": true,
   "dependencies": {
-    "@measured/puck": "^0.9.0",
-    "@measured/puck-plugin-heading-analyzer": "^0.9.0",
+    "@measured/puck": "0.12.0-canary.53e3fc5",
+    "@measured/puck-plugin-heading-analyzer": "0.12.0-canary.53e3fc5",
```

Comment:
We should remember to hop back on mainline one these features have been released